src/microsoft/office/license/user/microsoft-office-license-user.controller.js

```diff
@@ -62,7 +54,7 @@ angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicens
         this.users = null;
 
         return this.license.getUsers(this.currentLicense)
-            .then((users) => { this.users = users; })
+            .then((users) => { this.users = users.map((user) => ({ user })); })
```

Comment:
```myUser.user``` is a bit weird, ```myUser.id``` would be probably better

Reply:
done